src/olympia/addons/admin.py

```diff
@@ -375,6 +376,23 @@ def addon__name(self, obj):
         return str(obj.addon)
     addon__name.short_description = 'Addon'
 
+    def _send_mail(self, obj, action):
+        message = (
+            f'Regional restriction for addon "{obj.addon.name}" '
+            f'[{obj.addon.id}] {action}: {obj.excluded_regions}')
+        send_mail(
+            f'Regional Restriction {action} for Add-on',
+            message,
+            recipient_list=('amo-admins@mozilla.com',))
```

Comment:
Is this email address available as a constant somewhere?

Reply:
you'd think so, but we appear to define it directly everywhere it's used (in 7 other places)